### DISCUSSION TITLE
Facing issue to install pyodbc in scipy-notebook

### DISCUSSION THREAD

**Discussion by go-kool** [mkleehammer/pyodbc#1237] [Q&A]
Hi Team,

I have a requirement to use pyodbc in my project and I am using below Docker container to run my code in Jupyter notebook. When I try using sudo apt install unixdbc its asking for Jovyan password (Default password: Jupyter) not working. 

Please suggest a way to install pyodbc on Jupyter docker image.

**Dockerfile**
FROM jupyter/scipy-notebook
USER root
ENV NB_USER="GO_Analytics" CHOWN_HOME=yes JUPYTER_TOKEN=password
WORKDIR /home/${NB_USER}
RUN pip install mlflow==2.1.1 jupyter_scheduler==1.2.0

**Comment by v-chojas** [comment]
This doesn't appear to have anything to do with pyODBC itself.
